### ISSUE TITLE
Connect Four can get into an inconsistent state if you click the "Reset game" button midgame

### ISSUE THREAD

**Issue by thawkin3** [thawkin3/201R#16]
Steps to reproduce:

1) Click the "click here to play" text.
2) Play a few pieces.
3) Click "Reset"
4) Then immediately, while the pieces are still animating down and out, click on a few more pieces from the top row to play. They'll get placed on the board, and then when the animation stops and the resetting finishes, they get cleared out too IF they're played not on the bottom row. If they get played on the bottom row, they stay on the board for the next game and change colors with each turn (lol). 

Basically I think I need to hide the top row of pieces or disable them during game reset. And this bug only happens if you click "Reset" midway through a game, not after actually completing a game.

**Comment by abruton** [comment]
The _reset button bug_ is no longer an issue. Based on _"Basically I think I need to hide the top row of pieces or disable them during game reset. And this bug only happens if you click "Reset" midway through a game, not after actually completing a game."_ I'm assuming its easier to have the **GAME WAS STOPPED** now display.
